Added examples for progress control

FabienLelaquais opened the pull request:
+ removed 'color' property from progress control
+ added CSS class names for value and title in the control
+ minor refman issues

FabienLelaquais committed:
Added examples for progress control
+ removed 'color' property from progress control
+ added CSS class names for value and title in the control
+ minor refman issues

FredLL-Avaiga commented on taipy/gui/_renderers/factory.py:
```diff
@@ -444,6 +444,24 @@ class _Factory:
+                ("width", PropertyType.string_or_number),
+            ]
+        )
+        ._set_propagate(),
```
don't think `propagate` is needed here as there's no variable modification from the frontend

FabienLelaquais replied:
Let's discuss this rapidly. I'm sure you're right.

FabienLelaquais replied:
Done.

namnguyen20999 reviewed (dismissed):
lgtm

FabienLelaquais committed:
- Fred comments
- Rename element documentation example directory names to make them module (mypy)

FabienLelaquais committed:
Exclude builder doc examples from mypy file list

FabienLelaquais committed:
Exlude path fixed (?)

FabienLelaquais committed:
mypy again

FabienLelaquais committed:
mypy...

FabienLelaquais committed:
mypy...

FabienLelaquais committed:
Merge branch 'develop' into doc/taipy#1464-doc-for-progress-control

FabienLelaquais committed:
mypy...

FabienLelaquais committed:
Merge branch 'doc/taipy#1464-doc-for-progress-control' of github.com:Avaiga/taipy into doc/taipy#1464-doc-for-progress-control

FabienLelaquais committed:
Merge branch 'develop' into doc/taipy#1464-doc-for-progress-control
